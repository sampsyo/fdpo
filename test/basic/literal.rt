in a: 8;
out one: 8;
out two: 8;
out three: 8;
one = 8d27;
two = 8x9a;
three = 8b1011;
