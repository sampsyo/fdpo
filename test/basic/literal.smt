(let ((.def_0 (= three #b00001011))) (let ((.def_1 (= two #b10011010))) (let ((.def_2 (= one #b00011011))) (let ((.def_3 (and .def_2 .def_1 .def_0))) .def_3))))
