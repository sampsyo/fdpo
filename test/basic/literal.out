one = 27
two = 154
three = 11
